exporter/influxdbexporter/README.md

```diff
@@ -0,0 +1,141 @@
+
+### Definitions
+
+An OpenTelemetry ["Signal"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#opentelemetry-client-architecture) is general term for instances of categories of telemetry.
+For example:
+- Trace spans
+- Metrics
+- Logs
+
+An OpenTelemetry Signal ["Attribute"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes) to be a key-value pair whose key is a non-null, non-empty string, and whose value is not null and of type string, boolean, float64, int64, or homogeneous arrays of the previously mentioned primitives.
+Attribute names should follow the [naming guidelines](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-and-label-naming.md).
+For example:
+- `service.name`
+- `host.name`
+- `cloud.region`
+- `org_id`
+- `env`
+
+An OpenTelemetry Signal Label is similar to a Signal Attribute.
+
+An OpenTelemetry Signal ["Resource"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md) Attribute is "an immutable representation of the entity producing telemetry".
+In other words, the entity that generated the Signal.
+For example:
+- Host machine or VM
+- Kubernetes node, pod, and container
+- Application
+
+An OpenTelemetry Signal ["Instrumentation Library"](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md#instrumentation-libraries) Attribute is "a library that enables OpenTelemetry observability for another library".
+In other words, the library used by the host application to generate the Signal.
+For example:
+- The Go OpenTelemetry implementation
+- The Rust OpenTelemetry implementation
+
+Signal Attributes are also defined by the application emitting the Signal.
+
+Signals also have properties that are not free-form Attributes or Labels.
+These properties distinguish the OpenTelemetry Signal types.
+For example:
+- trace ID, span ID, parent span ID
+- metric name
+- log severity, log span ID
+
+### InfluxDB Conversion from OpenTelemetry
+
+Spans are stored in measurement `spans`.
+Metric points are assigned measurement from the `Metric.name` protocol buffer field.
+Logs are stored in measurement `logs`.
+
+This exporter converts Attributes to InfluxDB fields, verbatim.
+In the case that application Attribute keys conflict with Resource or Instrumentation Library Attribute keys, the application loses.
+
+The exporter converts other Signal properties to fields with key names borrowed from [OTLP protocol buffer](https://github.com/open-telemetry/opentelemetry-proto) messages.
+For example:
+- `Span.start_time_unix_nano` (type fixed64) -> InfluxDB line protocol timestamp
+- `Span.trace_id` (type bytes) -> `trace_id` as hexadecimal string
+- `Span.name` (type string) -> `name`
+- `Span.end_time_unix_nano` (type fixed64) -> `end_time_unix_nano` as uint64
```

Comment:
I don't know much about InfluxDb but if it allows analysis by fields then sending the duration (i.e. end_time_unix_nano - start_time_unix_nano) may be more interesting.